statistics.go

```diff
@@ -59,6 +60,6 @@ func SizeToString(size uint64) (str string) {
 
 // String returns human-readable Stats
 func (s Stats) String() string {
-	return fmt.Sprintf("%d files, %d dirs, %d symlinks, %d errors, %v Original Size, %v Storage Size",
-		s.Files, s.Dirs, s.SymLinks, s.Errors, SizeToString(s.Size), SizeToString(s.StorageSize))
+	return fmt.Sprintf("%d files, %d dirs, %d symlinks, %d errors, %v Original Size, %v Storage Size, %d Included Symlink-Files which are not backed up",
```

Comment:
Same here: I think we should add this between `%d symlinks` and `%d errors` as `%d excluded`.